src/coreclr/tools/Common/Compiler/NativeAotNameMangler.cs

```diff
@@ -69,7 +64,7 @@ public override string SanitizeName(string s, bool typeName = false)
                 sb ??= new StringBuilder(s, 0, i, s.Length);
 
                 // For CppCodeGen, replace "." (C# namespace separator) with "::" (C++ namespace separator)
-                if (typeName && c == '.' && _mangleForCplusPlus)
```

Comment:
Shouldn't this whole if be removed ? In other places you remove code using the assumption that _mangleForCplusPlus is always false but now code in this if will be executed when it wasn't before.

(I don't have any context here so this change might be right. I just saw the change and I found it weird.)

Reply:
Yes, would be nice to delete the comment above too and other references to C++ in this file

Reply:
You are right. Thanks for noticing!